tests/test-dirs/locate-type.t/run.t

```diff
@@ -1,3 +1,8 @@
+  $ ocamlc b.ml a.ml -bin-annot -o a.cmi
+  $ cat >.merlin <<EOF
+  > S .
+  > B .
+  > EOF
```

Comment:
`.` is added to the source and build dirs by default. So this `.merlin` is useless (or could remain empty).